### PR TITLE
feat(api): default token / group

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      POSTGRES_USER: cds
+
+  cds-cache:
+    image: redis:alpine
+    command: redis-server --requirepass cds
+    ports:
+      - "6379:6379"
+
+  cds-migrate:
+    image: ovhcom/cds-api:0.7.0
```
Comment (fsamin) [inline docker-compose.yml]: use latest tag on all our images

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      CDS_DB_USER: cds
+      CDS_DB_NAME: cds
+      CDS_DB_PORT: 5432
+      CDS_DB_SSLMODE: disable
+    links:
+       - cds-db
+
+  cds-api:
+    image: ovhcom/cds-api:0.7.0
+    command: bash -c "sleep 10 ; /app/api-linux-amd64"
```
Comment (fsamin) [inline docker-compose.yml]: why bash -c ?
remove sleep 10

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    volumes:
+      - cds-artefacts-volume:/app/artefacts
+    environment:
+      CDS_VCS_REPOSITORIES_GITHUB_STATUSES_URL_DISABLED: "true"
```
Comment (fsamin) [inline docker-compose.yml]: remove quotes

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      CDS_VCS_REPOSITORIES_BITBUCKET_STATUSES_DISABLED: "true"
+      CDS_DB_HOST: cds-db
+      CDS_DB_PASSWORD: cds
+      CDS_DB_TIMEOUT: 10000
+      CDS_DB_USER: cds
+      CDS_DB_NAME: cds
+      CDS_DB_MAXCONN: 40
+      CDS_DB_PORT: 5432
+      CDS_DB_SSLMODE: disable
+      CDS_URL_API: localhost:8081
```
Comment (fsamin) [inline docker-compose.yml]: humm, this won't work

File: engine/api/README.md [hunk 5]
```diff
@@ -260,69 +336,3 @@ disabled = false #This is mainly for dev purpose, you should not have to change
     statuses_disabled = false
     privatekey = "" # You can define here your bickcket private key if you don't use secret-backend-manager
 ```
-
-## Environment Variable Configuration
```
Comment (fsamin) [inline engine/api/README.md]: ?

File: engine/api/bootstrap/bootstrap.go [hunk 6]
```diff
@@ -1,40 +1,63 @@
...
 	"github.com/ovh/cds/engine/api/environment"
 	"github.com/ovh/cds/engine/api/group"
+	"github.com/ovh/cds/engine/api/worker"
+	"github.com/ovh/cds/sdk"
 	"github.com/ovh/cds/sdk/log"
 )
 
 //InitiliazeDB inits the database
-func InitiliazeDB(DBFunc func() *gorp.DbMap) error {
+func InitiliazeDB(defaultGroupName string, sharedInfraToken string, DBFunc func() *gorp.DbMap) error {
```
Comment (fsamin) [inline engine/api/bootstrap/bootstrap.go]: a bit ugly, this should be optionnal

File: docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  cds-cache:
+    image: redis:alpine
+    command: redis-server --requirepass cds
+    ports:
+      - "6379:6379"
+
+  cds-migrate:
+    image: ovhcom/cds-api:latest
+    command: /app/api-linux-amd64 database upgrade --db-host cds-db --db-user cds --db-password cds --db-name cds --db-sslmode disable  --migrate-dir /app/sql
+    environment:
```
Comment (sguiheux) [inline docker-compose.yml]: delete environment

File: engine/api/README.md [hunk 8]
```diff
@@ -166,7 +234,8 @@ timeout = 3000
 #mode = "redis"
 mode = "local"
 ttl = 60
-	# Connect CDS to a redis cache If you more than one CDS instance and to avoid losing data at startup
+
+    # Connect CDS to a redis cache If you more than one CDS instance and to avoid losing data at startup
     [cache.redis]
     host = "localhost:6379" # If your want to use a redis-sentinel based cluster, follow this syntax ! <clustername>@sentinel1:26379,sentinel2:26379sentinel3:26379
     password = "your password"
```
Comment (fsamin) [inline engine/api/README.md]: default password in docker-compose should be the same as default password in template file

File: engine/api/README.md (hunk 5 above)
Comment (yesnault) [inline engine/api/README.md]: keep above

File: docker-compose.yml (hunk 3 above)
Comment (yesnault) [inline docker-compose.yml]: not possible in compose file

**Pull request merged**